cmd/api-response.go

```diff
@@ -597,9 +612,12 @@ func generateListObjectsV2Response(bucket, prefix, token, nextToken, startAfter,
 	data.IsTruncated = isTruncated
 
 	commonPrefixes := make([]CommonPrefix, 0, len(prefixes))
-	for _, prefix := range prefixes {
+	for _, cprefix := range prefixes {
 		var prefixItem = CommonPrefix{}
-		prefixItem.Prefix = s3EncodeName(prefix, encodingType)
+		prefixItem.Prefix = s3EncodeName(cprefix, encodingType)
+		if cprefix == slashSeparator {
```

Comment:
s/cprefix/prefix ?

Reply:
it is `cprefix` @vadmeste since that is the `for loop`